testing/kuttl/e2e/root-cert-ownership/05--check-secret.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # If there are other PostgresClusters in the namespace, ensure that 'owner2'
+  # and 'owner2' are not listed.
+  # If there are no other PostgresClusters in the namespace, the 'pgo-root-cacert'
+  # secret should be deleted.
+  - script: |
+      NUM_CLUSTERS=$(kubectl --namespace="${NAMESPACE}" get postgrescluster --output name | wc -l)
+      ROOT_SECRET=$(kubectl --namespace="${NAMESPACE}" get secret pgo-root-cacert --output name | wc -l)
+      if [[ "$NUM_CLUSTERS" != 0 ]]; then
+          CURRENT_OWNERS=$(kubectl --namespace="${NAMESPACE}" get secret \
+          pgo-root-cacert -o jsonpath='{.metadata.ownerReferences[*].name}')
+          echo "$CURRENT_OWNERS"
+          if [[ "$CURRENT_OWNERS" == *"owner1"* ]]; then
+              exit 1
+          fi
+          if [[ "$CURRENT_OWNERS" == *"owner1"* ]]; then
+              exit 1
+          fi
+      else
+          if [[ "$ROOT_SECRET" != 0 ]]; then
```

Comment:
Since we only use ROOT_SECRET here, I wonder if you could move line 11's kubectl call into this `else` loop. (You could put it just as it is above line 23 -- usually if I'm just using a var once, I'll want to cut it out, but for readability, `[[ "$ROOT_SECRET" != 0 ]]` might be preferable to ``[[ $(kubectl --namespace="${NAMESPACE}" get secret pgo-root-cacert --output name | wc -l) != 0 ]]`!

Reply:
Agreed on moving to the 'else'. I originally just grouped this with `NUM_CLUSTERS` to make it a bit cleaner, but it technically doesn't need to be called for the 'else' case, so it's worth moving. Also agreed on not removing ROOT_SECRET all together, I find that statement a bit tricky to parse at first glance.